docs/source/installing-graphql-tools.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Using a different version of graphql-tools
+---
+
+Apollo Server includes `graphql-tools` version 4 by default. If you want to use a newer version, you can do so with the following steps:
+
+1. Install `graphql-tools` separately in your project.
+
+2. Update your `ApolloServer` constructor to provide the `schema` option _instead of_ `typeDefs`, `resolvers`, and `schemaDirectives`. You pass these options to the `makeExecutableSchema` function, which you provide as the value of `schema`:
```

Comment:
Unfortunately this is problematic as `schema` also has a few other minor effects — it leaves out the cache control directives (which weirdly are added unconditionally even if you pass `cacheControl: false`) and the uploads integration. I think it's at least worth maybe documenting that if you do this you'll have to include the cache control enum and directive declaration yourself?

I'm also considering fixing this by letting plugins add things to the schema like cacheControl does so it would just work always instead of being affected by this argument... Probably an AS3 thing though...